### ISSUE TITLE
ClassGraph#verbose() causes FileSystemNotFoundException exception with 4.8.0

### ISSUE THREAD

**Issue by robfitz273k** [classgraph/classgraph#316]
With the following file, VerboseTest.java.

```java
public class VerboseTest {
    public static void main(String[] args) {
        new io.github.classgraph.ClassGraph().verbose().scan();
    }
}
```

Using the classgraph-4.8.0.jar pulled from maven, at runtime the a FileSystemNotFoundException is triggered.

```
$ javac -cp classgraph-4.8.0.jar VerboseTest.java
$ java -cp classgraph-4.8.0.jar:. VerboseTest
Exception in thread "main" java.nio.file.FileSystemNotFoundException
        at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getFileSystem(ZipFileSystemProvider.java:169)
        at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getPath(ZipFileSystemProvider.java:155)
        at java.base/java.nio.file.Path.of(Path.java:208)
        at java.base/java.nio.file.Paths.get(Paths.java:97)
        at nonapi.io.github.classgraph.utils.VersionFinder.getVersion(VersionFinder.java:185)
        at nonapi.io.github.classgraph.utils.LogNode.<init>(LogNode.java:139)
        at io.github.classgraph.ClassGraph.verbose(ClassGraph.java:107)
        at VerboseTest.main(VerboseTest.java:3)
```

Removing the verbose() the call the code runs correctly.  The FileSystemNotFoundException occurs with both Java 11 and Java 8.

In my case in VersionFinder.java line 185, classpathResource  = jar:file:/home/bob/NetBeansProjects/VerboseTest/classgraph-4.8.0.jar!/io/github/classgraph/ClassGraph.class
Looks like FileSystemNotFoundException needs to be added to the ignored exceptions in VersionFinder.java, missed in 54f80ced00d473d04f6c7edf5235e1432aaf4df4.

Using the classgraph-4.6.32.jar pulled from maven, the correct output is generated. 

```
$ javac -cp classgraph-4.6.32.jar VerboseTest.java
$ java -cp classgraph-4.6.32.jar:. VerboseTest
Feb 14, 2019 12:40:48 PM nonapi.io.github.classgraph.utils.LogNode flush
INFO: 
Feb 14, 2019 12:41:37 P.M. nonapi.io.github.classgraph.utils.LogNode flush
INFO: 2019-02-14T12:41:37.685+0000	ClassGraph	ClassGraph version 4.6.32
2019-02-14T12:41:37.686+0000	ClassGraph	Operating system: Linux 5.0.0-rc4-00059-g62967898789d amd64
2019-02-14T12:41:37.686+0000	ClassGraph	Java version: 11.0.2 / 11.0.2+9-Debian-3 (Oracle Corporation)
2019-02-14T12:41:37.686+0000	ClassGraph	Java home: /usr/lib/jvm/java-11-openjdk-amd64
2019-02-14T12:41:37.697+0000	ClassGraph	ScanSpec:
2019-02-14T12:41:37.697+0000	ClassGraph	-- packageWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- packagePrefixWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- pathWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- pathPrefixWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- classWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- classfilePathWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- classPackageWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- classPackagePathWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- moduleWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- jarWhiteBlackList: 
2019-02-14T12:41:37.698+0000	ClassGraph	-- classpathElementResourcePathWhiteBlackList: 
2019-02-14T12:41:37.699+0000	ClassGraph	-- libOrExtJarWhiteBlackList: 
2019-02-14T12:41:37.699+0000	ClassGraph	-- performScan: true
2019-02-14T12:41:37.699+0000	ClassGraph	-- scanJars: true
2019-02-14T12:41:37.699+0000	ClassGraph	-- scanNestedJars: true
2019-02-14T12:41:37.699+0000	ClassGraph	-- scanDirs: true
2019-02-14T12:41:37.699+0000	ClassGraph	-- scanModules: true
2019-02-14T12:41:37.699+0000	ClassGraph	-- enableClassInfo: false
2019-02-14T12:41:37.699+0000	ClassGraph	-- enableFieldInfo: false
2019-02-14T12:41:37.699+0000	ClassGraph	-- enableMethodInfo: false
2019-02-14T12:41:37.699+0000	ClassGraph	-- enableAnnotationInfo: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- enableStaticFinalFieldConstantInitializerValues: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- enableInterClassDependencies: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- enableExternalClasses: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- enableSystemJarsAndModules: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- ignoreClassVisibility: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- ignoreFieldVisibility: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- ignoreMethodVisibility: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- disableRuntimeInvisibleAnnotations: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- extendScanningUpwardsToExternalClasses: true
2019-02-14T12:41:37.700+0000	ClassGraph	-- enableRemoteJarScanning: false
2019-02-14T12:41:37.700+0000	ClassGraph	-- addedClassLoaders: null
2019-02-14T12:41:37.701+0000	ClassGraph	-- overrideClassLoaders: null
2019-02-14T12:41:37.701+0000	ClassGraph	-- addedModuleLayers: null
2019-02-14T12:41:37.701+0000	ClassGraph	-- overrideModuleLayers: null
2019-02-14T12:41:37.701+0000	ClassGraph	-- overrideClasspath: null
2019-02-14T12:41:37.701+0000	ClassGraph	-- classpathElementFilters: null
2019-02-14T12:41:37.701+0000	ClassGraph	-- initializeLoadedClasses: false
2019-02-14T12:41:37.701+0000	ClassGraph	-- removeTemporaryFilesAfterScan: false
2019-02-14T12:41:37.701+0000	ClassGraph	-- ignoreParentClassLoaders: false
2019-02-14T12:41:37.701+0000	ClassGraph	-- ignoreParentModuleLayers: false
2019-02-14T12:41:37.701+0000	ClassGraph	-- modulePathInfo: 
2019-02-14T12:41:37.702+0000	ClassGraph	Finding classpath entries
2019-02-14T12:41:37.710+0000	ClassGraph	-- Finding classpath and modules
2019-02-14T12:41:37.710+0000	ClassGraph	---- System jars:
2019-02-14T12:41:37.710+0000	ClassGraph	------ Scanning disabled for lib or ext jar: /usr/lib/jvm/java-11-openjdk-amd64/lib/jrt-fs.jar
2019-02-14T12:41:37.767+0000	ClassGraph	---- Found ClassLoaders:
2019-02-14T12:41:37.767+0000	ClassGraph	------ jdk.internal.loader.ClassLoaders$AppClassLoader
2019-02-14T12:41:37.767+0000	ClassGraph	---- Found system modules:
2019-02-14T12:41:37.767+0000	ClassGraph	------ [module java.base, location=jrt:/java.base]
2019-02-14T12:41:37.767+0000	ClassGraph	------ [module java.compiler, location=jrt:/java.compiler]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.datatransfer, location=jrt:/java.datatransfer]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.desktop, location=jrt:/java.desktop]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.instrument, location=jrt:/java.instrument]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.logging, location=jrt:/java.logging]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.management, location=jrt:/java.management]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.management.rmi, location=jrt:/java.management.rmi]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.naming, location=jrt:/java.naming]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.net.http, location=jrt:/java.net.http]
2019-02-14T12:41:37.768+0000	ClassGraph	------ [module java.prefs, location=jrt:/java.prefs]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.rmi, location=jrt:/java.rmi]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.scripting, location=jrt:/java.scripting]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.security.jgss, location=jrt:/java.security.jgss]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.security.sasl, location=jrt:/java.security.sasl]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.smartcardio, location=jrt:/java.smartcardio]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.sql, location=jrt:/java.sql]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.sql.rowset, location=jrt:/java.sql.rowset]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.transaction.xa, location=jrt:/java.transaction.xa]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.xml, location=jrt:/java.xml]
2019-02-14T12:41:37.769+0000	ClassGraph	------ [module java.xml.crypto, location=jrt:/java.xml.crypto]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.accessibility, location=jrt:/jdk.accessibility]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.attach, location=jrt:/jdk.attach]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.charsets, location=jrt:/jdk.charsets]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.compiler, location=jrt:/jdk.compiler]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.crypto.cryptoki, location=jrt:/jdk.crypto.cryptoki]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.crypto.ec, location=jrt:/jdk.crypto.ec]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.dynalink, location=jrt:/jdk.dynalink]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.editpad, location=jrt:/jdk.editpad]
2019-02-14T12:41:37.770+0000	ClassGraph	------ [module jdk.httpserver, location=jrt:/jdk.httpserver]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.internal.ed, location=jrt:/jdk.internal.ed]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.internal.jvmstat, location=jrt:/jdk.internal.jvmstat]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.internal.le, location=jrt:/jdk.internal.le]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.internal.opt, location=jrt:/jdk.internal.opt]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.jartool, location=jrt:/jdk.jartool]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.javadoc, location=jrt:/jdk.javadoc]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.jconsole, location=jrt:/jdk.jconsole]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.jdeps, location=jrt:/jdk.jdeps]
2019-02-14T12:41:37.771+0000	ClassGraph	------ [module jdk.jdi, location=jrt:/jdk.jdi]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.jdwp.agent, location=jrt:/jdk.jdwp.agent]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.jfr, location=jrt:/jdk.jfr]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.jlink, location=jrt:/jdk.jlink]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.jshell, location=jrt:/jdk.jshell]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.jsobject, location=jrt:/jdk.jsobject]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.jstatd, location=jrt:/jdk.jstatd]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.localedata, location=jrt:/jdk.localedata]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.management, location=jrt:/jdk.management]
2019-02-14T12:41:37.772+0000	ClassGraph	------ [module jdk.management.agent, location=jrt:/jdk.management.agent]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.management.jfr, location=jrt:/jdk.management.jfr]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.naming.dns, location=jrt:/jdk.naming.dns]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.naming.rmi, location=jrt:/jdk.naming.rmi]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.net, location=jrt:/jdk.net]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.scripting.nashorn, location=jrt:/jdk.scripting.nashorn]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.sctp, location=jrt:/jdk.sctp]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.security.auth, location=jrt:/jdk.security.auth]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.security.jgss, location=jrt:/jdk.security.jgss]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.unsupported, location=jrt:/jdk.unsupported]
2019-02-14T12:41:37.773+0000	ClassGraph	------ [module jdk.unsupported.desktop, location=jrt:/jdk.unsupported.desktop]
2019-02-14T12:41:37.774+0000	ClassGraph	------ [module jdk.xml.dom, location=jrt:/jdk.xml.dom]
2019-02-14T12:41:37.774+0000	ClassGraph	------ [module jdk.zipfs, location=jrt:/jdk.zipfs]
2019-02-14T12:41:37.774+0000	ClassGraph	---- Found non-system modules:
2019-02-14T12:41:37.774+0000	ClassGraph	------ [None]
2019-02-14T12:41:37.775+0000	ClassGraph	---- ClassLoaderHandlers:
2019-02-14T12:41:37.784+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.AntClassLoaderHandler
2019-02-14T12:41:37.784+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.EquinoxClassLoaderHandler
2019-02-14T12:41:37.784+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.EquinoxContextFinderClassLoaderHandler
2019-02-14T12:41:37.784+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.FelixClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.JBossClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.WeblogicClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.WebsphereLibertyClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.WebsphereTraditionalClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.OSGiDefaultClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.SpringBootRestartClassLoaderHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.TomcatWebappClassLoaderBaseHandler
2019-02-14T12:41:37.785+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.ParentLastDelegationOrderTestClassLoaderHandler
2019-02-14T12:41:37.786+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.JPMSClassLoaderHandler
2019-02-14T12:41:37.786+0000	ClassGraph	------ nonapi.io.github.classgraph.classloaderhandler.URLClassLoaderHandler
2019-02-14T12:41:37.786+0000	ClassGraph	---- ClassLoader jdk.internal.loader.ClassLoaders$AppClassLoader will be handled by nonapi.io.github.classgraph.classloaderhandler.JPMSClassLoaderHandler@31e863bd
2019-02-14T12:41:37.787+0000	ClassGraph	---- jdk.internal.loader.ClassLoaders$AppClassLoader delegates to parent jdk.internal.loader.ClassLoaders$PlatformClassLoader with order PARENT_FIRST
2019-02-14T12:41:37.787+0000	ClassGraph	---- ClassLoader jdk.internal.loader.ClassLoaders$PlatformClassLoader will be handled by nonapi.io.github.classgraph.classloaderhandler.JPMSClassLoaderHandler@7c0835c3
2019-02-14T12:41:37.788+0000	ClassGraph	---- Finding classpath elements in ClassLoaders
2019-02-14T12:41:37.788+0000	ClassGraph	------ Finding classpath elements in ClassLoader jdk.internal.loader.ClassLoaders$PlatformClassLoader
2019-02-14T12:41:37.788+0000	ClassGraph	------ Finding classpath elements in ClassLoader jdk.internal.loader.ClassLoaders$AppClassLoader
2019-02-14T12:41:37.788+0000	ClassGraph	---- Getting classpath entries from java.class.path
2019-02-14T12:41:37.789+0000	ClassGraph	------ Found classpath element: classgraph-4.6.32.jar -> /home/bob/NetBeansProjects/VerboseTest/classgraph-4.6.32.jar
2019-02-14T12:41:37.789+0000	ClassGraph	------ Found classpath element: . -> /home/bob/NetBeansProjects/VerboseTest
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.base
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.compiler
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.datatransfer
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.desktop
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.instrument
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.logging
2019-02-14T12:41:37.789+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.management
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.management.rmi
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.naming
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.net.http
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.prefs
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.rmi
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.scripting
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.security.jgss
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.security.sasl
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.smartcardio
2019-02-14T12:41:37.790+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.sql
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.sql.rowset
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.transaction.xa
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.xml
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: java.xml.crypto
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.accessibility
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.attach
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.charsets
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.compiler
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.crypto.cryptoki
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.crypto.ec
2019-02-14T12:41:37.791+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.dynalink
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.editpad
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.httpserver
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.internal.ed
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.internal.jvmstat
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.internal.le
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.internal.opt
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jartool
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.javadoc
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jconsole
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jdeps
2019-02-14T12:41:37.792+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jdi
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jdwp.agent
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jfr
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jlink
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jshell
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jsobject
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.jstatd
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.localedata
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.management
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.management.agent
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.management.jfr
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.naming.dns
2019-02-14T12:41:37.793+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.naming.rmi
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.net
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.scripting.nashorn
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.sctp
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.security.auth
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.security.jgss
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.unsupported
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.unsupported.desktop
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.xml.dom
2019-02-14T12:41:37.794+0000	ClassGraph	-- Skipping non-whitelisted or blacklisted system module: jdk.zipfs
2019-02-14T12:41:37.797+0000	ClassGraph	-- Reading jarfile directories and manifest files (took 0.041658 sec)
2019-02-14T12:41:37.806+0000	ClassGraph	---- Opening jar: /home/bob/NetBeansProjects/VerboseTest/classgraph-4.6.32.jar
2019-02-14T12:41:37.820+0000	ClassGraph	------ This is a multi-release jar
2019-02-14T12:41:37.827+0000	ClassGraph	-- Final classpath element order:
2019-02-14T12:41:37.827+0000	ClassGraph	---- /home/bob/NetBeansProjects/VerboseTest/classgraph-4.6.32.jar
2019-02-14T12:41:37.827+0000	ClassGraph	---- /home/bob/NetBeansProjects/VerboseTest
2019-02-14T12:41:37.827+0000	ClassGraph	-- Scanning filenames within classpath elements
2019-02-14T12:41:37.828+0000	ClassGraph	---- Scanning directory classpath element /home/bob/NetBeansProjects/VerboseTest (took 0.005754 sec)
2019-02-14T12:41:37.834+0000	ClassGraph	------ No whitelisted classfiles found
2019-02-14T12:41:37.829+0000	ClassGraph	------ Scanning directory: /home/bob/NetBeansProjects/VerboseTest (took 0.004607 sec)
2019-02-14T12:41:37.831+0000	ClassGraph	-------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/build (took 0.000766 sec)
2019-02-14T12:41:37.831+0000	ClassGraph	---------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/build/classes
2019-02-14T12:41:37.831+0000	ClassGraph	---------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/build/empty
2019-02-14T12:41:37.832+0000	ClassGraph	---------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/build/generated-sources (took 0.000160 sec)
2019-02-14T12:41:37.832+0000	ClassGraph	------------ Scanning directory: /home/bob/NetBeansProjects/VerboseTest/build/generated-sources/ap-source-output
2019-02-14T12:41:37.831+0000	ClassGraph	---------- Found whitelisted resource:  build/built-jar.properties
2019-02-14T12:41:37.832+0000	ClassGraph	-------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/dist (took 0.000634 sec)
2019-02-14T12:41:37.832+0000	ClassGraph	---------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/dist/lib
2019-02-14T12:41:37.833+0000	ClassGraph	------------ Found whitelisted resource:  dist/lib/classgraph-4.6.32.jar
2019-02-14T12:41:37.832+0000	ClassGraph	---------- Found whitelisted resource:  dist/README.TXT
2019-02-14T12:41:37.832+0000	ClassGraph	---------- Found whitelisted resource:  dist/VerboseTest.jar
2019-02-14T12:41:37.833+0000	ClassGraph	-------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/nbproject (took 0.000785 sec)
2019-02-14T12:41:37.833+0000	ClassGraph	---------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/nbproject/private
2019-02-14T12:41:37.833+0000	ClassGraph	------------ Found whitelisted resource:  nbproject/private/config.properties
2019-02-14T12:41:37.834+0000	ClassGraph	------------ Found whitelisted resource:  nbproject/private/private.properties
2019-02-14T12:41:37.833+0000	ClassGraph	---------- Found whitelisted resource:  nbproject/build-impl.xml
2019-02-14T12:41:37.833+0000	ClassGraph	---------- Found whitelisted resource:  nbproject/genfiles.properties
2019-02-14T12:41:37.833+0000	ClassGraph	---------- Found whitelisted resource:  nbproject/project.properties
2019-02-14T12:41:37.833+0000	ClassGraph	---------- Found whitelisted resource:  nbproject/project.xml
2019-02-14T12:41:37.834+0000	ClassGraph	-------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/src
2019-02-14T12:41:37.834+0000	ClassGraph	-------- Scanning directory: /home/bob/NetBeansProjects/VerboseTest/test
2019-02-14T12:41:37.830+0000	ClassGraph	-------- Found whitelisted resource:  VerboseTest.java
2019-02-14T12:41:37.830+0000	ClassGraph	-------- Found whitelisted resource:  build.xml
2019-02-14T12:41:37.830+0000	ClassGraph	-------- Found whitelisted resource:  classgraph-4.6.32.jar
2019-02-14T12:41:37.830+0000	ClassGraph	-------- Found whitelisted resource:  classgraph-4.8.0.jar
2019-02-14T12:41:37.831+0000	ClassGraph	-------- Found whitelisted resource:  manifest.mf
2019-02-14T12:41:37.831+0000	ClassGraph	-------- Found whitelisted resource:  out
2019-02-14T12:41:37.828+0000	ClassGraph	---- Scanning jarfile classpath element /home/bob/NetBeansProjects/VerboseTest/classgraph-4.6.32.jar (took 0.003176 sec)
2019-02-14T12:41:37.831+0000	ClassGraph	------ No whitelisted classfiles found
2019-02-14T12:41:37.830+0000	ClassGraph	------ Found whitelisted resource:  META-INF/MANIFEST.MF
2019-02-14T12:41:37.830+0000	ClassGraph	------ Found whitelisted resource:  META-INF/maven/io.github.classgraph/classgraph/pom.properties
2019-02-14T12:41:37.830+0000	ClassGraph	------ Found whitelisted resource:  META-INF/maven/io.github.classgraph/classgraph/pom.xml
2019-02-14T12:41:37.839+0000	ClassGraph	Masking classfiles
2019-02-14T12:41:37.840+0000	ClassGraph	Classfile scanning is disabled
2019-02-14T12:41:37.845+0000	ClassGraph	Completed (took 0.135954 sec)

Feb 14, 2019 12:41:37 P.M. nonapi.io.github.classgraph.utils.LogNode flush
INFO: 
```

**Comment by lukehutch** [comment]
@robfitz273k thank you for the very thorough report! Most of the static analyzers I used I used to get the 4.8.0 release ready don't like generic exceptions, so I replaced them with concrete exceptions, but it made me nervous that I may have missed some cases like this. I think I'm going to go back through and replace catch blocks like this with generic exceptions again.

Should be fixed in master. I'll push a release out later today.
